src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+info:
+    description: "Inventory Management application"
+spring:
+  application:
+    name: inventory-service
+  data:
+    mongodb:
+        uri: mongodb://localhost:27017/OMS

```

Comment:
port , uri: mongodb://localhost:27017/OMS  needs to be removed .

Reply:
OK :). it will be removed. I have raised the PR so that I can pull this from home